index.html

```diff
@@ -0,0 +1,135 @@
+                </div>
+              </div>
+            </section>
+        </div>
+      </div>
+    </section>
+    
+      <div class="main">
+        <div class="container">
+          <div class="main__inf">
+      <section class="my_goal">
+        <h2>My goal</h2>
+        <ol>
+          <li>
+            I want to get a new profession! I want to become high-quality
+            programmer.
+          </li>
+          <li>I want to raise living standards of my family.</li>
+          <li>
+            I want to study programming on JavaScript language to achieve my
+            goals.
+          </li>
+        </ol>
+      </section>
+      <section class="my_skills">
+        <h2>My skills</h2>
+        <p><b>Programming languages:</b></p>
+        <ul>
+          <li>JavaScript: beginner level</li>
+          <li>HTML&CSS: beginner level</li>
+          <li>Skill to work in team</li>
+          <li>And a very big desire</li>
+        </ul>
+      </section>
+      <div class="various__skills">
+          <section class="experience">
+            <h2>Experience</h2>
+            <ul>
+              <li>
+                <a href="https://www.codecademy.com/profiles/rm.levsha87)"
+                  >account Codecademy</a
+                >
+              </li>
+              <li>
+                <a href="https://www.codewars.com/users/levsha87" )
+                  >account Codewars</a
+                >
+              </li>
+              <li><a href="https://www.github.com/levsha87">account github</a></li>
+            </ul>
+          </section>
+          <section class="education">
+            <h2>Education</h2>
+            <p>
+              Military Academy of the Republic of Belarus (faculty of interia guard
+              troops) 2005-2010
+            </p>
+          </section>
+          <section class="level_eng">
+            <h2>English</h2>
+            <p>
+              My English level is A1, soon will be A2. I learn English myself. I
+              will study English at Stremline courses!
+            </p>
+          </section>
+        </div>
+        <section class="code__example">
+            <h2>Code examples (tasks with Codewars): JavaScript</h2>
+            <pre>
+              function getDivisorsCnt(n){
+                      let i =1;
+                      let k = 0;
+                      while (i<=n){
+                          if (n%i==0){
+                              k = k+1;
+                              i=i+1;
+                          } else {
+                              i=i+1;
+                          }
+                      }
+                      return k;
+                      } 
+              </pre>
+        </section>
+        </div>
+      </div>
+    </div>                  
+  </body>
+  <footer>
+    <div class="container">
+      <div class="author__name">
+    <p>Copyright Roman Levsha</p> <a href="#">rm.levsha@gmail.com</a>
```

Comment:
<p> doesn't fit from a semantic perspective, as it's meant for multiple-line text (paragraph).
I recommend using an inline element. - https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements
Also, you can use a copyright sign here (mnemonic symbol) 
![image](https://user-images.githubusercontent.com/8201843/92738202-6b522680-f384-11ea-9108-5616e2dbb8bd.png)